compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1047,6 +1047,18 @@ static SmallVector<int64_t> getLinalgExtDefaultWorkgroupTileSizes(
       workgroupTileSizes[dim] = 0;
     }
   }
+
+  OpBuilder builder(op.getContext());
```

Comment:
Uggh! I know why we need this, I wish we didnt.... We probably need a `getStaticIterationDomain` method in `TilingInterface`....

Edit: Could we move this part into the `setRootConfig` of the `unpack` op. Then a method to unpack op to get the static iteration domain can avoid using the builder?

Reply:
This is not needed anymore. The tiling implementation should handle one-trip loop natively. It is not a good fix-up to me because these one-trip loops should be folded away by canonicalization patterns.